prov/gni/src/gnix_sep.c

```diff
@@ -119,6 +119,11 @@ static int gnix_sep_tx_ctx(struct fid_ep *sep, int index,
 		(index >= sep_priv->info->ep_attr->tx_ctx_cnt))
 		return -FI_EINVAL;
 
+	if (attr && attr->op_flags & ~GNIX_EP_OP_FLAGS) {
```

Comment:
If that's the case, we should probably return -FI_EINVALID.

I was reading the op_flags section:
Flags  that  control the operation of operations submitted against the context.
Maybe we should add verbage like the caps description:
The capabilities must be a subset of those requested of  the  associated  endpoint.